### ISSUE TITLE
What will the final form of my predictions be?

### ISSUE THREAD

**Issue by olivboom** [msc-acse/acse-9-independent-research-project-OliverJBoom#17]
Should probably be log returns, but very difficult to debug these and harder to see if it's learning.

**Comment by olivboom** [comment]
When using log returns the network overfits to training dataset very easily.

But when using scaled prices the network generalises better

**Comment by olivboom** [comment]
Use raw prices. Calculations on log returns don't work and calculations on inverse prices are a poor work-around. 
 
    -    Mention in report reasons for log returns poor performance
             -    Highly non-linear
             -    Crash in the middle of the dataset
             -    MSE is the evaluation metric and so safest option is a flat signal. 
                   Optimises towards this.
